spec.html

```diff
@@ -422,22 +422,31 @@ <h1>Runtime Semantics: Evaluation</h1>
         <emu-alg>
```

Comment:
I don't think any of these changes belongs in this proposal (unless there's some necessary change I'm not seeing). Spec refactors would be better as a editorial change on ecma262 repo.

Reply:
Which changes did you have in mind?

There are a couple of refactors, but only with the express purpose of reuse among new clauses. For example, EvaluateDynamicPropertyAccess is used in the runtime semantics for the following production.

> *OptionalChain* : *OptionalChain* `[` *Expression* `]`</emu-grammar>

Similarly, EvaluateStaticPropertyAccess is used in the runtime semantics for

> *OptionalChain* : `?.` *IdentifierName*

Reply:
Ahhh, now it makes more sense. 👍, let's keep this refactoring.